blocks/bricks/language_models.py

```diff
@@ -0,0 +1,81 @@
+from blocks.bricks import Feedforward, Initializable, Tanh, Linear, application
+from blocks.bricks.recurrent import SimpleRecurrent
+from blocks.bricks.parallel import Fork
+
+
+class LanguageModel(Feedforward, Initializable):
```

Comment:
Why not use `SequenceGenerator` for that? The [markov chain demo](https://github.com/mila-udem/blocks-examples/blob/master/markov_chain/__init__.py) shows how it can be used for sequence modelling.